src/entrypoint/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"regexp"
+)
+
+type Response struct {
+	ProviderId          string `json:"providerId"`
+	ProviderName        string `json:"providerName"`
+	ProviderDisplayName string `json:"providerDisplayName"`
+	UrlSyncUX           string `json:"urlSyncUX"`
+	UrlAPI              string `json:"urlAPI"`
+	Width               int    `json:"width"`
+	Height              int    `json:"height"`
+}
+
+func main() {
+	http.HandleFunc("/", handleEntryPoint)
+
+	log.Fatal(http.ListenAndServe(":8080", nil))
+}
+
+func handleEntryPoint(w http.ResponseWriter, r *http.Request) {
+	err, host, port := parseUrl(r.URL.Path)
+	if err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	entryPointUrl := fmt.Sprintf(`https://%v:%v/modules/domain-connect/index.php/`, host, port)
+	response := Response{
+		ProviderId:          "plesk",
```

Comment:
It makes sense to extract "plesk", "Plesk", "Domain ... " and 750 into package-level constants.
This would also allow to not duplicate this values in test.